Source/WarpX.H

```diff
@@ -877,6 +877,24 @@ private:
     // Domain decomposition on Level 0
     amrex::IntVect numprocs{0};
 
+    //
+    // Embeded Boundary
+    //
+
+    // Factory for field data
+    amrex::Vector<std::unique_ptr<amrex::FabFactory<amrex::FArrayBox> > > m_factory;
+
+    amrex::FabFactory<amrex::FArrayBox> const& Factory (int lev) const noexcept {
```

Comment:
Factory is a very general term in programming for a function that generates classes.
As a geeneral `WarpX::Factory` method it seems a bit technical/non-descript. Do you have a suggestion how we could make it more descriptive?

E.g. `make_fields` and `make_EBfields` or so?

Reply:
I like. `fieldFactory`.  I will make the change.